### PR TITLE
Adds Phase 1 Exercise 2a input files

### LINKED ISSUE #52: Add Phase 1 Exercise 2a input files to the repo
This issue can be closed when the necessary files to reproduce the results are added to the repo.
Those files should include:
* Moltres input files
* Geometry files
* .csv files necessary for obtaining plots and results
* Necessary functions for carrying out the post-processing

### PR DESCRIPTION
**Pull request by robfairh**
Fixes #52 

The input files include two models:
* first model uses INL benchmark model
* second model uses a sub-channel approach for obtaining the fuel and moderator temperatures

### CODE REVIEW

File: benchmark-ph1ex2/README.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# Phase 1 Exercise 2:
+
+* Thermal-fluids steady-state.
+* exercise defines a core power distribution map in OECD-MHTGR350-power.inp which can be found in the ARFC's box.
+* 4 sub-cases:
+* 2a: no bypass, fixed thermophysical properties
+* 2b: bypass I, fixed thermophysical properties
+* 2c/d: bypass I and II, variable thermophysical properties
```
Comment (smpark7) [inline benchmark-ph1ex2/README.md:8]: I and II refer to coolant channels 1 and 2?

File: benchmark-ph1ex2/coupled-files/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* simple.i: global model
+* unit.i: sub-channel model
+
+To run them do:
+(alias runmoltres='conda activate moose; mpirun -np 12 $MOLTRES/moltres-opt -i')
+runmoltres unit.i
+
+Results:
+--------
+Figure temp_x=0.png is the temperature on the left boundary.
```
Comment (smpark7) [inline benchmark-ph1ex2/coupled-files/README.md:13]: How does the radial temperature profile look? Does it have any undershoots/overshoots around the gap interfaces?

**Review by smpark7 — COMMENTED** [review]
Your moltres input files look good. Did you create the 2nd model yourself or is there a reference? It looks quite different compared to the INL model.

I'm interested to see the radial temperature profile in your unit cells because there are so few mesh elements in the radial direction and the gap has such a small k value.

**Comment by robfairh** [comment]
@smpark7 this is INL's geometry:

image

and this is ``` model1.png ```:

model1

INL uses a sub-channel approach [link](https://inldigitallibrary.inl.gov/sites/sti/sti/6007188.pdf). The sub-channel approach calculates the coolant temperatures in the global model and, then calculates the fuel and moderator temperature in the unit cell model. My results of the global model - i.e my coolant temperatures - are already off and I didn't calculate the temperature in the unit cell.

My second model is based on Stainsby's approach [link](https://www.nrc.gov/docs/ML0909/ML090900018.pdf). Which models the HTGR core hexagonal fuel elements and defines a single coolant channel in the middle of the fuel element. My model adds the simplification of translating that into 2D, where the hexagonal fuel elements become cylinders and the single coolant channel in the middle of the fuel elements becomes a single coolant ring in the middle of the first fuel ring.
This is why ``` model2.png ``` contains 3 coolant rings.

model2

File: benchmark-ph1ex2/README.md (hunk 1 above)
Comment (robfairh) [reply benchmark-ph1ex2/README.md:8]: The readme contains information on all the exercise 2 subcases. My model included only Exercise 2a. I and II refer to the bypass types I and II defined [here](https://www.oecd-nea.org/jcms/pl_19834/nea-benchmark-of-the-modular-high-temperature-gas-cooled-reactor-350-mw-core-design-volumes-i-and-ii).

**Comment by robfairh** [comment]
Here is the radial profile in the unit cell:

image

To reproduce this result it is necessary to:
* create global mesh: open ``` 2D-fullcore-model2.geo ``` with gmsh and create 2D mesh
* run global model input file: ```mpirun -np 4 $MOLTRES/moltres-opt -i input-model2.i```
* create unit cell mesh: open ``` unitcell-layers.geo ``` with gmsh and create 3D mesh
* run unit cell model input file: ```mpirun -np 4 $MOLTRES/moltres-opt -i unitcell-R1.i```
* visualize 3D results in paraview and plot temperature over the line: (0,0,596.5)-(1.62813,-0.94,596.5)

**Comment by robfairh** [comment]
The radial profile from ```coupled-files/unit.i``` is the following:

image

**Comment by robfairh** [comment]
The second model requires to run two input files for each unit cell: ```input-model2.i``` and ```unitcell-R1.i```.
```coupled-files``` contains some files that prove that it is possible to obtain the second model results by running only one input file.

**Comment by smpark7** [comment]
Looks good! Thanks for clarifying the differences in the model and your temperature profile.

**Review by smpark7 — APPROVED** [review]

**Pull request merged**